### ISSUE TITLE
Browser compatibility issue with Split extension

### ISSUE THREAD

**Issue by namrathaj** [snuziale/vsts-extension-split-work#20]
The Split functionality doesn't work in Chrome browser. 

P S It works with Internet explorer

Looking forward for the fix of this issue at the earliest

**Comment by adamtoakley** [comment]
I'm having this issue as well

**Comment by RicardoLans** [comment]
Same here, it does work when using IE, that is my workaround for now.
Please fix this for Chrome use.

**Comment by beccamc** [comment]
I need more information about the problem with Chrome. 

1. Does the extension load in the context menu? If not, please open developer tools (F12) and take a screenshot of any errors you see. Look for key words "blueprint", "vsts-extension-split-work", or "split"
2. If the extension loads in the context menu, can you open the first split dialog? Does it fail when you click the "Split" button? Are there errors or does it just do nothing?
3. What version of chrome are you using? You can find this in "Help" -> "About Google Chrome"

Thanks!

**Comment by RicardoLans** [comment]
![image](https://user-images.githubusercontent.com/10512755/55378315-c7224a00-5518-11e9-8be0-0d2919b5c50f.png)

Extension does load but when clicking Split it just closes the dialog and nothing happens, checked dev tools and also no console logging nor network event is happening.

I am using the latest version of Chrome: Version 73.0.3683.86 (Official Build) (64-bit)

**Comment by RicardoLans** [comment]
Also i have noticed in IE that sometimes the dialog is not working.
In IE I did see a network HTTP Post 400 bad request with the following body:
```
{
    "$id": "1",
    "customProperties": {
        "FieldReferenceName": null,
        "FieldStatusFlags": "none",
        "ErrorMessage": "The field 'Assigned To' contains the value 'Ricardo Lans' that is not in the list of supported values",
        "FieldStatusCode": 0,
        "RuleValidationErrors": [{
            "fieldReferenceName": "System.AssignedTo",
            "fieldStatusFlags": "hasValues, limitedToValues, allowsOldValue, invalidListValue",
            "errorMessage": "The field 'Assigned To' contains the value 'Ricardo Lans' that is not in the list of supported values",
            "fieldStatusCode": 4194380,
            "ruleValidationErrors": null
        }]
    },
    "innerException": null,
    "message": "The field 'Assigned To' contains the value 'Ricardo Lans' that is not in the list of supported values",
    "typeName": "Microsoft.TeamFoundation.WorkItemTracking.Server.RuleValidationException, Microsoft.TeamFoundation.WorkItemTracking.Server",
    "typeKey": "RuleValidationException",
    "errorCode": 0,
    "eventId": 3200
}
```

I dont know why I see this error message cause I got 2 logins in my own DevOps (personal account and work account) and both accounts are valid contributors in that project.

**Comment by sghsmorgan** [comment]
Having same issue with Chrome

**Comment by TroySchmidt** [comment]
Cannot use this either because of chrome issues.  Randomly it will work so I feel like it is a timing issue. But it works maybe 1 in 25 tries.

**Comment by davidcorrigan714** [comment]
Just submitted a pull request that should resolve these issues. Hopefully it gets picked up.

**Comment by RicardoLans** [comment]
@beccamch could you check on the PR (#23) that is mentioned?

**Comment by beccamc** [comment]
Yep I saw! Thanks David! I will try to look today

**Comment by beccamc** [comment]
This should be fixed as of version 1.4.  Big thanks to @davidcorrigan714 for fixing this!
